test/src/module_test.dart

```diff
@@ -76,6 +76,22 @@ void main() {
 
       expect(find.text('test'), findsOneWidget);
     });
+    testWidgets('Module.getCurrentLoadModule retrieves module instance from context', (tester) async {
+      await ModulesManager.instance.initModules([TestModule()]);
+      await tester.pumpWidget(MaterialApp(
+        home: ModuleWidget<TestModule>(
+          child: Builder(
+            builder: (context) {
+              final Module? module = Module.getCurrentLoadModule(context);
+              final TestModule testingModule = (module as TestModule);
+              return Text(testingModule.moduleName);
+            },
+          ),
+        ),
+      ));
+
+      expect(find.text('TestModule'), findsOneWidget);
+    });
```

Comment:
To make sure it is really getting the same module instance, I think it would be better to test like that:
```suggestion
    testWidgets('Module.of retrieves module instance from context', (tester) async {
      final testModule = TestModule();
      await ModulesManager.instance.initModules([testModule]);
      await tester.pumpWidget(
        ModuleWidget<TestModule>(
          child: Builder(
            builder: (context) {
              return const SizedBox.shrink();
            },
          ),
        ),
      );
      final context = tester.element(find.byType(SizedBox));
      final module = Module.of(context);
      expect(module, equals(testModule));
    });
```